<launch>

  <node pkg= "tensorflow_object_detector" name="detect_ros" type="detect_ros.py"  output="screen"> 
    <remap from="image" to="/usb_cam_node/image_raw"/>
	</node>

  <param name="video_path" type="string" value="/home/sun/2022-06-10 00-33-31.mp4" />

  <node pkg= "tensorflow_object_detector" name="video_to_camera" type="video_to_camera.py"  output="screen">
    <remap from="/usb_cam/image_raw" to="/usb_cam_node/image_raw"/>
    
  </node>

  <node pkg="image_view" type="image_view" name="image_view">
    <remap from="image" to="debug_image"/>
  </node>
  <node name="node2" pkg="hw_07" type="node2"/>

</launch>
